app/helpers/form_helpers.py

```diff
@@ -21,9 +22,17 @@ def sanitise_mobile_number(data: str) -> str:
     return re.sub(r"^(0{1,2}44|\+44|0)", "", data)
 
 
-def format_playback_value(value: float | Decimal, currency: str | None = None) -> str:
+def format_playback_value(
+    value: float | Decimal,
+    currency: str | None = None,
+    decimal_limit: int | None = None,
+) -> str:
+    # For playback of values we set the decimal limit based on the number of decimal places
+    # in the given value rather than any limit that has been set in the schema as we do for other methods
```

Comment:
Is it okay to use` """ """ `rather then `#` so the comment "flows" better when reading.

Reply:
Yep - good spot, have fixed the ones in the integration tests too